tests/tests/tracer.rs

```diff
@@ -160,7 +159,7 @@ async fn test_trace_block(#[future] plain_opcodes: (Katana, KakarotEvmContract),
     let trace_vec = block_traces.unwrap_or_default();
     // We expect 3 traces per transaction: CALL, CREATE, and CALL.
     // Except for the last one which is out of resources.
-    assert!(trace_vec.len() == 3 * (TRACING_TRANSACTIONS_COUNT - 1) + 1);
+    assert_eq!(trace_vec.len(), TRACING_TRANSACTIONS_COUNT);
```

Comment:
@greged93 Do we have something that changed here for all the other tests to pass and not this assertion? Seems like now we have only 1 CALL per transaction and no more CALL + CREATE + CALL, maybe this is normal with the relayer system as I cannot see anything that has changed for this specific test

Reply:
should still be the same, means the tracing is maybe failing :/